README.md

```diff
@@ -31,18 +31,21 @@ These instructions are written assuming Mac OS X install.
 ```
 # Homebrew - http://brew.sh/
 ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/homebrew/go/install)"
+brew update
 
 # Git - http://git-scm.com/
 brew install git
```

Comment:
Might be nice to move the dependencies to a [Brewfile](https://robots.thoughtbot.com/brewfile-a-gemfile-but-for-homebrew)